app/assets/javascripts/dashboard/spotlight_view.js

```diff
@@ -117,36 +115,26 @@ type Props = {
 
 type State = {
   datasets: Array<APIMaybeUnimportedDatasetType>,
+  hasOrganisations: boolean,
   isLoading: boolean,
 };
 
 class SpotlightView extends React.PureComponent<Props, State> {
   state = {
     datasets: [],
+    hasOrganisations: true,
     isLoading: true,
   };
 
   componentDidMount() {
     this.fetchData();
-    this.maybeRedirectToOnboarding();
-  }
-
-  async maybeRedirectToOnboarding() {
-    const organizations = await getOrganizations();
-    if (organizations.length > 0) {
-      return;
-    }
-
-    message.info(messages["setup.redirect"]);
-    await Utils.sleep(2000);
-    this.props.history.push("/onboarding");
   }
 
   async fetchData(): Promise<void> {
     try {
       this.setState({ isLoading: true });
-      const datasets = await getDatasets();
-      this.setState({ datasets });
+      const [datasets, organizations] = await Promise.all([getDatasets(), getOrganizations()]);
+      this.setState({ datasets, hasOrganisations: organizations.length > 0 });
```

Comment:
I'd suggest to also use the american english spelling for `organization` here :)